packages/components/src/components/hds/app-header/home-link.hbs

```diff
@@ -14,7 +14,14 @@
   @href={{@href}}
   @isHrefExternal={{@isHrefExternal}}
   ...attributes
-  aria-label={{this.ariaLabel}}
+  aria-label={{if this.isIconOnly this.text null}}
 >
   <Hds::Icon @name={{@icon}} @color={{@color}} @stretched={{true}} />
+  {{#unless this.isIconOnly}}
```

Comment:
This isn't exactly the same as the conditional logic for the `<Hds::Button>` but it seemed to make sense given that the icon is always required. But open to suggestions!

Reply:
This makes sense to me as well.